### PR TITLE
Updated the content to use the new InstallFilebeat template.

### PR DESCRIPTION
**Pull request by ChrisAdams-Logit**
https://logitio.monday.com/boards/806674751/pulses/7531384957

This is an update so that the content uses the new InstallFilebeat template. The template was created in this PR:

https://github.com/logit-io/docs/pull/85

### CODE REVIEW

File: src/pages/integrations/log-management/applications/activemq.mdx [hunk 1]
```diff
@@ -16,7 +16,7 @@ Configure Filebeat using the pre-defined examples below to start sending and ana
   <InstallIntegration/>
   ### Install Filebeat
 
-  <InstallFilebeat />
+  <InstallFilebeat items={["DEB", "RPM"]} />
```
Comment (ccottam) [inline src/pages/integrations/log-management/applications/activemq.mdx:20]: I think activemq is java, so all os options could be supported, although I doubt any one does

File: src/pages/integrations/log-management/applications/haproxy.mdx [hunk 2]
```diff
@@ -15,7 +15,7 @@ Configure Filebeat to ship logs from HAProxy to Logstash and Elasticsearch.
   <InstallIntegration/>
   ### Install Filebeat
 
-  <InstallFilebeat />
```
Comment (ccottam) [inline src/pages/integrations/log-management/applications/haproxy.mdx:18]: I think all os options could be supported, although I doubt any one does

File: src/pages/integrations/log-management/operating-systems/centos.mdx [hunk 3]
```diff
@@ -17,7 +17,7 @@ Configure Filebeat to ship logs from Centos Systems to Logstash and Elasticsearc
 
   ### Install Filebeat
 
-  <InstallFilebeat />
```
Comment (ccottam) [inline src/pages/integrations/log-management/operating-systems/centos.mdx:20]: centos probably just wants the rpm option

File: src/pages/integrations/log-management/operating-systems/redhat.mdx [hunk 4]
```diff
@@ -16,7 +16,7 @@ Configure Filebeat to ship logs from Red Hat Systems to Logstash and Elasticsear
   <InstallIntegration/>
   ### Install Filebeat
 
-  <InstallFilebeat />
```
Comment (ccottam) [inline src/pages/integrations/log-management/operating-systems/redhat.mdx:19]: just the rpm option

File: src/pages/integrations/log-management/operating-systems/ubuntu.mdx [hunk 5]
```diff
@@ -16,7 +16,7 @@ Configure Filebeat to ship logs from Ubuntu Systems to Logstash and Elasticsearc
   <InstallIntegration/>
   ### Install Filebeat
 
-  <InstallFilebeat />
```
Comment (ccottam) [inline src/pages/integrations/log-management/operating-systems/ubuntu.mdx:19]: just the deb option

**Review by ccottam — CHANGES_REQUESTED** [review]

**Comment by ccottam** [comment]
@ChrisAdams-Logit changes requested - Also worth making sure the same options are in any other steps in the same doc

File: src/pages/integrations/log-management/applications/haproxy.mdx (hunk 2 above)
Comment (ccottam) [reply src/pages/integrations/log-management/applications/haproxy.mdx:18]: ignore this - looked a bit harder - looks like windows support is more docker/cygwin

**Review by leesmith110 — COMMENTED** [review]
Accepted Changes

File: src/pages/integrations/log-management/applications/activemq.mdx (hunk 1 above)
Comment (leesmith110) [reply src/pages/integrations/log-management/applications/activemq.mdx:20]: For now we will leave as is, until we review again soon

**Review by leesmith110 — APPROVED** [review]

**Pull request merged**